charts/flagsmith/templates/_helpers.tpl

```diff
@@ -202,6 +202,28 @@ Influxdb hostname
 {{ template "flagsmith.influxdb.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 {{- end -}}
 
+{{/*
+Frontend environment
+*/}}
+{{- define "flagsmith.frontend.environment" -}}
+- name: ASSET_URL
+  value: '/'
+{{- if .Values.frontend.apiProxy.enabled }}
+- name: PROXY_API_URL
```

Comment:
We can remove the definition of PROXY_API_URL now. It's a hangover from people using older versions of our image but, particularly for the tests, we don't care about that, right? Therefore, we only need the `FLAGSMITH_PROXY_API_URL` defined.

Reply:
Just seen the next file and noticed that this is used in the deployment template too. BUT, this value was changed in september 2021... I think we can safely remove the `PROXY_API_URL` definition now.